### PR TITLE
Upgraded to Go 1.9 and dep

### PR DESCRIPTION
**Pull request by febbraro**
Move away from Godep to dep (thanks to @sdboyer)

This is for #68

### CODE REVIEW

File: cli/Gopkg.lock [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  name = "golang.org/x/sys"
+  packages = ["unix"]
+  revision = "eb2c74142fd19a79b3f237334c7384d5167b1b46"
+
+[[projects]]
+  name = "gopkg.in/yaml.v2"
+  packages = ["."]
+  revision = "cd8b52f8269e0feb286dfeef29f8fe4d5b397e0b"
+
+[solve-meta]
```
Comment (grayside) [inline cli/Gopkg.lock]: Not sure where it belongs, but depending readme recommends ci should use a fixed version.

Comment (grayside) [reply cli/Gopkg.lock]: `dep` readme, not depending

**Comment by febbraro** [comment]
@grayside Updated based on feedback. Multiple dependencies did not have explicit versions, so I pinned to a revision

**Comment by sdboyer** [comment]
hola friends! couple quick notes on this:

1. you'll probably have an easier time if you use release versions of dep instead of tip.
2. if there's ever a possibility that something else will depend on rig, you'll likely want to place the `Gopkg.{toml,lock},vendor` at the repository root.
3. in general, you needn't specify a `revision` in `Gopkg.toml` - instead, trust `Gopkg.lock` to pin things appropriately. if there's no tags to consume, prefer a `branch`. then just make sure to always use `dep ensure -update <deps...>` instead of `dep ensure -update` (globally), and those pinned versions won't change.

**Comment by febbraro** [comment]
Thanks @sdboyer! Appreciate the feedback

**Comment by febbraro** [comment]
Moving all this work to #95

**Pull request closed without merging**